packages/terra-hookshot/src/_HookshotUtils.js

```diff
@@ -12,6 +12,21 @@ const MIRROR_TB = {
   bottom: 'top',
 };
 
+/**
+ * This function returns a rect 2px x 2px with the origin based at (1,1), allow for evenly centered attachment points.
+ * As a result of the 2x2 size, bounds checking errors as a result of rounding are eliminated.
+ *
+ * @ param {object} coords - THe x and y client coordinates.
+ */
+const getRectFromCoords = coords => ({
```

Comment:
Is there a check somewhere to verify that these coordinates are valid with the defined viewport?

Reply:
Technically as long as they are evaluated by PropTypes as numbers the coordinates are valid. While the use-case for out of range values is more limited, I don't see number reason to prevent it if someone wants an element slightly off screen.